include/revng/PipelineC/Prototypes.h

```diff
@@ -364,7 +364,8 @@ bool rp_manager_container_deserialize(rp_manager *manager,
                                       rp_step *step,
                                       const char *container_name,
                                       const char *content,
-                                      uint64_t size);
+                                      uint64_t size,
+                                      rp_invalidations *invalidations);
```

Comment:
We should really get rid of this primitive. It's... impure.
One day we'll have hashes in the model and we'll be able to get rid of it.

No action needed.

Reply:
I agree, this change is kind of in that direction because it introduces the same invalidation mechanism as analyses/diffs, so that once it does become one it will be transparent w.r.t. revng-ui.